_posts/2017-03-03-fantas-eel-and-specification.md

```diff
@@ -89,12 +89,12 @@ Shape.prototype.translate =
     })
   }
 
-Square(Coord(2, 2, 0), Coord(3, 3, 0))
-.translate(3, 3, 3)
+Shape.Square(Coord(2, 2, 0), Coord(3, 3, 0))
+    .translate(3, 3, 3)
```

Comment:
Used the actual constructors here via `Shape`: I know you go on to explain the diff between the type and the constructor, but the example doesn't run as-is without a destructuring of `Shape` into the constructors.

Rather than introduce another line with the destructuring I've just been explicit.